lms/static/sass/lms-footer.scss

```diff
@@ -0,0 +1,24 @@
+// Footer for OpenEdX (left-to-right)
+// ==================================
+
+// libs and resets *do not edit*
+@import 'bourbon/bourbon'; // lib - bourbon
+@import 'vendor/bi-app/bi-app-ltr'; // set the layout for left to right languages
+
+// base - utilities
+@import 'base/variables';
+@import 'base/mixins';
+
+//<THEME-OVERRIDE>
```

Comment:
Why is this comment here? The override happens in a separate Sass compile that's in the independent theme dir.